scrapy/utils/url.py

```diff
@@ -103,3 +103,16 @@ def guess_scheme(url):
         return any_to_uri(url)
     else:
         return add_http_if_no_scheme(url)
+
+
+def trim_href_attribute(href):
```

Comment:
I'd go for `strip_w3c_whitespace` then. With "html", one could understand stripping whitespace from HTML source code or something. (just my 2 cents)